source/_components/image_processing.openalpr_local.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+title: "OpenAlpr Local"
+description: "Instructions how to integrate licences plates with OpenAlpr local into Home Assistant."
+date: 2017-01-09 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: openalpr.png
+ha_category: Image_Processing
+featured: false
+ha_release: 0.36
+---
+
+[OpenAlpr](http://www.openalpr.com/) integration for Home Assistant allows you to process licences plates from a camera. You can use them to open a garage door or trigger any other [automation](https://home-assistant.io/components/automation/).
+
+### {% linkable_title Local installation %}
+
+If you want process all data local you need the command line tool `alpr` in version > 2.3.1
+
+If you don't find binaries for your distribution you can compile from source. Documention of how to build openalpr is found [here](https://github.com/openalpr/openalpr/wiki).
+
+On a debian system you can use this cmake command to build only the command line tool (which second part on linux build instruction - ubuntu 14.04+):
```

Comment:
"(which second part on linux build instruction - ubuntu 14.04+)" doesn't make sense to me